Rakefile

```diff
@@ -52,8 +58,13 @@ task :test do
           errors << "Link URL must have HTTP protocol for %s in %s." % [username, file]
         end
 
-        if mentor["link_text"].to_s.include?("Github")
-          errors << "GitHub should be spelled with an uppercase 'H' for %s in %s." % [username, file]
+        if linkText.downcase.gsub("https://","").gsub("http://","") == url.downcase.gsub("https://","").gsub("http://","")
```

Comment:
@kytrinyx if there is a cleaner way to go about this `if` I am all for it.  However, this is working.  All the bios that Travis CI is complaining about based on this latest version of Rakefile have been updated in #792.

Once we are happy with the way Rakefile is operating and all the necessary changes have been made to the bios I plan to remove `WIP` from #792.

Reply:
`gsub` can take a regex for the first argument:

```
link_text.gsub(/https?\:\/\//, "")
```

Reply:
I assumed that would be possible, my RegEx powers are weak.